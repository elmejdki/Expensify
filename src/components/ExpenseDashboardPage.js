import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const ExpenseDashboardPage = () => (
  <div>
    <ExpenseListFilters />
    <ExpenseList />
  </div>
);

export default ExpenseDashboardPage;