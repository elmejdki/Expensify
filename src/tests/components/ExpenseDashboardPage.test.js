import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

const renderer = new ShallowRenderer();

it('should render ExpenseDashboardPage correctly', () => {
  renderer.render(<ExpenseDashboardPage />);

  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
