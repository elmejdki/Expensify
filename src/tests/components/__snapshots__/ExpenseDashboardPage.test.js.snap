// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboardPage correctly 1`] = `
<div>
  <Memo(Connect(ExpensesSummary)) />
  <Memo(Connect(ExpenseListFilters)) />
  <Memo(Connect(ExpenseList)) />
</div>
`;
